src/concrete/ml/pytest/utils.py

```diff
@@ -184,6 +192,15 @@ def get_sklearn_linear_models_and_datasets(
             partial(TweedieRegressor, link="identity", power=0.0),
         ]
 
+    # If the linear model is SGDClassifier,
+    # we need to handle the training parameters
+    if is_model_class_in_a_list(SGDClassifier, linear_classes):
+        linear_classes += [
+            partial(SGDClassifier, fit_encrypted=False),
+            # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4460
+            # partial(SGDClassifier, fit_encrypted=True, parameters_range=(-1, 1)),
```

Comment:
why not activating it here ? does it cause issues ?

Reply:
I would like to activate it here but we need more work to make our test our custom SGDClassifier. It causes lots of issues yes. E.g. correctness.

Reply:
I see, thanks !